contracts/http/response.go

```diff
@@ -34,7 +34,7 @@ type ContextResponse interface {
 	// Status sets the HTTP response status code and returns the ResponseStatus.
 	Status(code int) ResponseStatus
 	// Stream sends a streaming response with the specified status code and the given reader.
-	Stream(step func(w StreamWriter) error, code int, contentType string) Response
+	Stream(step func(w StreamWriter) error, code int) Response
```

Comment:
Keep consistent with the `Json` method, `code` is the first paramter.

```suggestion
	Stream(code int, step func(w StreamWriter) error) Response
```

And add the `Stream` to `ResponseStatus`:

```
Stream(step func(w StreamWriter) error) Response
```

Reply:
Ok